diffusion/evaluation/clean_fid_eval.py

```diff
@@ -78,12 +80,14 @@ def __init__(self,
         self.batch_size = batch_size
         self.image_key = image_key
         self.caption_key = caption_key
+        self.load_strict_model_weights = load_strict_model_weights
```

Comment:
Can't we access this boolean from the trainer config later? Why bother storing a copy of it in the class?